quisp/modules/QRSA/RealTimeController/RealTimeController_test.cc

```diff
@@ -1,8 +1,19 @@
 #include "RealTimeController.h"
-#include "gtest/gtest.h"
+#include <gtest/gtest.h>
+#include <omnetpp.h>
+#include <utils/IComponentProviderStrategy.h>
 
 namespace {
 
+using namespace omnetpp;
+using namespace quisp::utils;
+using namespace quisp::modules;
+
+class TestComponentProviderStrategy : public IComponentProviderStrategy {
```

Comment:
We can change what the `ComponentProvider` returns by defining the class inherit`IComponentProviderStrategy` class.